webapp/src/App.css

```diff
@@ -1,7 +1,9 @@
 .App {
   text-align: center;
 }
-
+a {
+  text-decoration: none;
```

Comment:
I am a fan of how clean this is!

I'm not an expert in accessibility but just as an FYI: be aware some CSS styling like the dotted outline when you tab through things are sometimes there for a reason (not sure about this one in particular).

Accessibility requirements change from project-to-project and team-to-team, but imo it's something good to think about and learn about, even if it's an often under-appreciated skill

Reply:
Nice, I like it! Resolving this since there's no more to do